src/components/blocks/Column.tsx

```diff
@@ -0,0 +1,22 @@
+import { AttributesBlock } from 'types/AttributesBlock';
+import { UtkwdsColumnBlockAttributes } from 'client';
+import BlockRouter from 'components/BlockRouter';
+
+interface Props {
+  attributes: Partial<UtkwdsColumnBlockAttributes>;
+  innerBlocks?: AttributesBlock[];
+}
+
+const Column = ({
+  attributes: { className, colWidth },
+  innerBlocks,
+}: Props) => (
+  <div
+    className={`className col-12 col-md-${colWidth || ''} ${className || ''}`}
```

Comment:
The first `className` in this string is going to be part of the string-literal (because you didn't wrap it in `${}`), but in the `save.js` file in the Plugin repo it's a variable:

```js
export default function save( props ) {
  const { attributes, className } = props;

  return (
    <div className={ className + ' col-12 col-md-' + attributes.colWidth + ' ' + attributes.className }> 
      <InnerBlocks.Content />
    </div>
  );
}

```

Also, I think the inclusion of the `className` variable there is a _mistake_. There's a `props.attributes.className` (which is used at the end of the string, and which you've correctly duplicated in `Column.tsx`), but I don't think there's actually a `props.className`. I'm guessing we'll see something like `class="undefined col-12 ..."` if we look at the markup that results from that `save.js` file.

(I think you should just get rid of the `className` at the beginning of the string.)

Reply:
@MichaelAllenWarner ok – no problem. The plugin save.js file just had the className at the beginning of the string and then again as a variable call (I spun up a local WP to test and you are right both are returning undefined – see attached screenshot)
<img width="560" alt="wp-column-columns" src="https://user-images.githubusercontent.com/69162526/188728340-755cb40f-96ed-40bf-9bba-21f9e83752e6.png">